bin/utils/service.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+## Usage: ./idam-service-service.sh [microservice_name]
```

Comment:
The usage comment should match the script name

Also the script name should be meaningful, for example `get-idam-service-token.sh`

Reply:
It's my bad I will remove it